src/database/models/UserChallenge.ts

```diff
@@ -3,6 +3,10 @@ import { DataTypes } from 'sequelize';
 import Sequelize from '../connection';
 import { User } from './User';
 
+import { ChallengeType } from '~/shared/userChallenge';
+
+const ChallengeTypeValues = Object.values(ChallengeType);
```

Comment:
It`s better to store the constants in the specific module folder.
Sometimes we need it to reuse somewhere